.github/actions/spell-check/expect.txt

```diff
@@ -455,8 +475,11 @@ gacogne
 gatech
 Gavarret
 Gbps
+gcna
+gcovr
 gdpr
 Geijn
+generatec
```

Comment:
I know why it's there but we don't want to commit it like that :)

Reply:
After #14585 merges, I can drop it.

Reply:
Great, it has been merged a couple hours ago ;-)